terraform/aws-constellation/variables.tf

```diff
@@ -0,0 +1,76 @@
+variable "name" {
+  type        = string
+  description = "Name of your Constellation"
+}
+
+variable "image" {
+  type        = string
+  description = "Node image version. Needs to be a release version."
+} # is required resolve the AMI
```

Comment:
As stated above, we should allow to pass an image ref in the same format as the config uses here for compatibility with the current CI and to align with the config inputs. We can use the hack-tool `image-fetch` to get the actual CSP image ref from it.